kartothek/io_components/metapartition.py

```diff
@@ -97,7 +97,8 @@ def _initialize_store_for_metapartition(method, method_args, method_kwargs):
 
             if store_variable in args:
                 ix = args.index(store_variable)
-                # reduce index since the argspec and method_args start counting differently due to self
+                # reduce index since the argspec and method_args start counting
```

Comment:
where do these changes (similar in many other files) come from?

Reply:
This is due to  **black==19.10b0**  formatting. **format_code.sh** fromatted all these files.

Reply:
black doesn't format comments

Reply:
Reverted all the changes which are not part of this story.